src/DispatchListener.php

```diff
@@ -216,7 +216,7 @@ protected function marshalBadControllerEvent(
         $controllerName,
         MvcEvent $event,
         Application $application,
-        \Exception $exception
+        \Throwable $exception
```

Comment:
Missed that this was in the method signature and not the docblock. I'll remove the typehint during merge.